g9frontend/src/pages/UserPage/UserPage.tsx

```diff
@@ -0,0 +1,98 @@
+import React, {useState} from "react";
+import {Button, Container, ToggleButtonGroup} from "@mui/material";
+import ToggleButton from "../../components/ToggleButton/ToggleButton";
+import SendUsAMessageButtonComponent from "../../components/SendUsAMessageButton/SendUsAMessageButton";
+import {useParams} from "react-router-dom";
+import "./UserPageStyle.css";
+
+
+export const UserPage: React.FC = () => {
+    const [selectedPlatform, setSelectedPlatform] = useState<string | null>(null);
+    const {username} = useParams();
+
+    const styles = {
+        blueButton: {
+            width: 300,
+            height: 50,
+            backgroundColor: "#2196f3", /* Blue background color */
+            color: "#ffffff", /* White text color */
+            padding: "10 20",
+            border: "none",
+            borderRadius: 8,
+            fontSize: 16,
+            cursor: "pointer"
+        }
+    }
+
+    const handleSelect = (selectedButton: string): void => {
+        setSelectedPlatform(selectedButton);
+        console.log(username);
+    }
+
+    return (
+        <Container>
+            <div className="container">
+                <div className="header">
+                    <p className="logo">WiFi</p>
+                    <Button style={{
+                        textTransform: 'none',
+                        color: '#f7fefe',
+                        right: '3%',
+                        fontFamily: 'Inter, sans-serif',
+                        fontSize: '20px'
+                    }}>
+                        Log in
+                    </Button>
+                </div>
+                <div className="content">
+                    <div className="leftContainer">
+                        <div className="welcomeText">
+                            Welcome {username}!
+                        </div>
+                        <div className="centeredContainer">
+                            <Button className="blueButton" style={styles.blueButton}>
+                                Craft your post
+                            </Button>
```

Comment:
This button isn't centered and frankly, it's quite ugly. There is also a lot of empty space on the left side.